packages/react-test-renderer/README.md

```diff
@@ -1,12 +1,26 @@
-# `react-test-renderer`
+# `react-test-renderer` (DEPRECATED)
+
+## Deprecation notice
+
+`react-test-renderer` is deprecated and no longer maintained. It will be removed in a future version. As of React 19, you will see a console warning when invoking ReactTestRenderer.create().
```

Comment:
```suggestion
`react-test-renderer` is deprecated and no longer maintained. It will be removed in a future version. As of React 19, you will see a console warning when invoking `ReactTestRenderer.create()`.
```